notifier/libnotify.go

```diff
@@ -0,0 +1,29 @@
+package notifier
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func notifySend(text string, timeout time.Duration) error {
+	expireTime := fmt.Sprintf("--expire-time=%d", timeout/time.Millisecond)
+	return exec.Command("notify-send", text, expireTime).Run()
```

Comment:
How about we use [go-notify](https://github.com/mqu/go-notify) library to avoid spawning external processes?

Reply:
> How about we use go-notify library to avoid spawning external processes?

I don't think it's a good idea because it requires cgo.

Reply:
OK makes sense